src/components/ContactPage/index.tsx

```diff
@@ -17,6 +20,26 @@ export const ContactPage: FC = () => {
         ]}
         text="Like what you see? Let's get started."
       />
+      {/* Should this blurb be a component? Asking for a friend */}
+      <article
+        css={css`
+          margin-bottom: ${cssClamp([6.5, 'mobile'], [8.5, 'tablet'])};
+          margin-top: ${cssClamp([6.75, 'mobile'], [6.5, 'tablet'])};
+          position: relative;
+
+          ${atMinTablet} {
+            max-width: 52rem;
+          }
+        `}
+      >
+        <Text variant="h2">
+          Base Two is located in Columbus, OH, Pittsburgh, PA and Chicago, IL,
+          but thanks to satellites and the internet, we can work with you
+          wherever you are.
+        </Text>
+      </article>
```

Comment:
💭 I don't _think_ we need to turn this into a component? I'm not sure though, maybe there are other places where we use this same structure again?

On another topic entirely, I _think_ we might be able to combine the two nodes here using the polymorphic `as` prop to change the node type of `Text` from `p` to `article`. It's possible the styles would need to be adjusted as well if that's done though; just a thought. 🙃

```suggestion
      <Text
        as="article"
        css={css`
          margin-bottom: ${cssClamp([6.5, "mobile"], [8.5, "tablet"])};
          margin-top: ${cssClamp([6.75, "mobile"], [6.5, "tablet"])};
          position: relative;

          ${atMinTablet} {
            max-width: 52rem;
          }
        `}
        variant="h2"
      >
        Base Two is located in Columbus, OH, Pittsburgh, PA and Chicago, IL, but
        thanks to satellites and the internet, we can work with you wherever you
        are.
      </Text>
```